.vscode/settings.json

```diff
@@ -4,6 +4,9 @@
         "jmed",
         "Jooby",
         "nodenext",
-        "preinstall"
+        "OPTOFLASH",
+        "OPTOLOW",
+        "preinstall",
+        "REJOJN"
```

Comment:
misprint?

and it would be nice to keep this list sorted

Reply:
REJOJN - part of event name, already in constants

manage this file manually - available, but not required, so keep sorting - just a
 time waste

Reply:
ok
I'll move the question

Reply:
word deleted